### PR TITLE
Add Realization num to histogram rug plot hovering

### LINKED ISSUE #83: Add realisation number to the hovering of the data in parameter plot

### PR DESCRIPTION
**Pull request by autt**
Resolves #83

### CODE REVIEW

File: ertviz/models/plot_model.py [hunk 1]
```diff
@@ -134,10 +134,17 @@ def repr(self):
         colors = []
         data = []
         names = []
+        realization_num = []
```
Comment (xjules) [inline ertviz/models/plot_model.py:137]: Nice, was wondering if you could change the variable name to plural, eg. realization_nums, for the sake of clarity that we talk many.

Comment (autt) [reply ertviz/models/plot_model.py]: Sure!

File: ertviz/models/plot_model.py [hunk 2]
```diff
@@ -134,10 +134,17 @@ def repr(self):
         colors = []
         data = []
         names = []
+        realization_nums = []
         for response_name in self._data_df_dict:
             data.append(list(self._data_df_dict[response_name].values.flatten()))
             colors.append(self._colors[response_name])
             names.append(response_name)
+            realization_nums.append(
+                list(
```
Comment (dotfloat) [inline ertviz/models/plot_model.py:143]: `list(foo for foo in bar)` is the same as `[foo for foo in bar]`. The first is a generator comprehension given to the `list` function to make it into a `list`, the second is a list comprehension that does it for you and doesn't require an extra indent.

**Review by xjules — APPROVED** [review]
Good job, LGTM!

**Pull request merged**